apps/fortuna/src/chain/eth_gas_oracle.rs

```diff
@@ -61,93 +52,15 @@ where
     }
 
     async fn estimate_eip1559_fees(&self) -> Result<(U256, U256)> {
-        self.provider
-            .estimate_eip1559_fees(Some(eip1559_default_estimator))
+        let (max_fee_per_gas, max_priority_fee_per_gas) = self
+            .provider
+            .estimate_eip1559_fees(None)
```

Comment:
This line should still be using eip1559_default_estimator. Please undo the deletion of that here.